theme/tutorialCodeValidation.less

```diff
@@ -0,0 +1,77 @@
+/* Import all components */
+@import 'themes/default/globals/site.variables';
+@import 'themes/pxt/globals/site.variables';
+
+/* Reference import */
+@import (reference) "semantic.less";
+
+
+/*******************************
+    Tutorial Code Validation
+*******************************/
+
+.tutorialCodeValidation {
+    position: absolute;
+    max-width: 100%;
+    min-width: 300px;
+    top: ~'calc(50% + 3rem)';
+    right: 2rem;
+    padding: 10px 26px;
+    border-radius: 0.2rem;
+    background: @white;
+    z-index: @tutorialLightboxCardZIndex;
+    border: 8px solid @black;
```

Comment:
why black for this? maybe worth sending a screenshot to sonia for design feedback, or bringing to design office hours tomorrow